apps/lib/hooks/useKatanaAprs.ts

```diff
@@ -2,7 +2,7 @@ import axios from 'axios'
 import { useEffect, useState } from 'react'
 
 const CACHE_KEY = 'katana-aprs-cache'
-const CACHE_TTL = 5 * 60 * 1000 // 5 min
+const CACHE_TTL = 1 * 60 * 1000 // 1 min
```

Comment:
this could be 15 minutes because the latest cache update sets the apr service's cache to 15 min https://github.com/yearn/katana-apr-service/blob/e097e26f812b03a7ce5b75157211fa76fd6e9deb/src/app/api/vaults/route.ts#L20